CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
 ## 0.2.0
 
-* Added `Control.Applicative.Combinators.NonEmpty` module that exports
```

Comment:
We still have wrong changelog entry here. But do nothing now I'll handle it for you.